curvefs/src/client/common/config.h

```diff
@@ -66,6 +66,7 @@ struct ExcutorOpt {
     uint64_t minRetryTimesForceTimeoutBackoff = 5;
     uint64_t maxRetryTimesBeforeConsiderSuspend = 20;
     uint32_t batchLimit = 100;
+    bool enableRenameParallel = false;
```

Comment:
> I think the defalut value should be true.

I had set the default value of `fuseClient.enableMultiMountPointRename` to `true`, if we set the RPC configure item to true, it will cause confusion for people who invoke rpc, because we will get latest txid for every dentry related RPC 
 request when the  `enableRenameParallel ` to true.

Reply:
If someone not set 'fuseClient.enableMultiMountPointRename', the value will be false and error may occur when rename at multile mount point.

Reply:
> If someone not set 'fuseClient.enableMultiMountPointRename', the value will be false and error may occur when rename at multile mount point.

not yet, there is no effect for other RPC request.